creator-node/src/services/stateMachineManager/monitoring/StateMonitoringQueue.js

```diff
@@ -68,11 +71,22 @@ class StateMonitoringQueue {
         lockDuration: STATE_MONITORING_QUEUE_MAX_JOB_RUNTIME_MS,
         // We never want to re-process stalled jobs
         maxStalledCount: 0
+      },
+      limiter: {
+        // Bull doesn't allow either of these to be set to 0
+        max: config.get('stateMonitoringQueueRateLimitJobsPerInterval') || 1,
+        duration: config.get('stateMonitoringQueueRateLimitInterval') || 1
       }
     })
   }
 
-  registerQueueEventHandlers(queue) {
+  /**
+   * Registers event handlers for logging and job success/failure.
+   * @param {Object} queue the queue to register events for
+   * @param {Function<queue, successfulJob, jobResult>} jobSuccessCallback the function to call when a job succeeds
+   * @param {Function<queue, failedJob>} jobFailureCallback the function to call when a job fails
+   */
+  registerQueueEventHandlers(queue, jobSuccessCallback, jobFailureCallback) {
```

Comment:
this is way cleaner imo
very nitty - this one would be more readable with named params (can do later)

Reply:
👍 sneaking that change into a future PR